### PR TITLE
Add key binding for fetching key and re-verifying message

### PR DESCRIPTION
**Pull request by rakoo**

### CODE REVIEW

File: lib/sup/crypto.rb [hunk 1]
```diff
@@ -16,6 +16,9 @@ class Error < StandardError; end
     [:encrypt, "Encrypt only"]
   )
 
+  KEY_PATTERN = /(-----BEGIN PGP PUBLIC KEY BLOCK.*-----END PGP PUBLIC KEY BLOCK)/m
+  KEYSERVER_URL = "http://pool.sks-keyservers.net:11371/pks/lookup"
```
Comment (maxmeyer) [inline lib/sup/crypto.rb]: Hello @rakoo,

does it make sense to make the `KEYSERVER_URL` configurable via sup-config-yaml?

Cheers,
mm

Comment (rakoo) [inline lib/sup/crypto.rb]: Hello @maxmeyer 

I don't know about that. To be honest I have no idea what keyservers are available out there, all I know is that this is a pool so you should be able to find the key you're looking for.

**Comment by rakoo** [comment]
@maxmeyer I added the possibility to configure the url through the `:keyserver_url` key in the config. By default the pool url is used.

**Comment by gauteh** [comment]
:+1: 

Excerpts from Matthieu Rakotojaona's message of January 28, 2015 22:11:
> @maxmeyer I added the possibility to configure the url through the `:keyserver_url` key in the config. By default the pool url is used.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/sup-heliotrope/sup/pull/387#issuecomment-71916780
=

**Comment by rakoo** [comment]
Thanks guys, merged !

**Comment by maxmeyer** [comment]
Great! :+1:

**Pull request merged**